smrf/envphys/radiation.py

```diff
@@ -1070,8 +1071,16 @@ def solint(a, b):
     # calcualte splines
     c = Akima1DInterpolator(wave, val)
 
-    # Take the integral between the two wavelengths
-    intgrl, ierror = quad(c, a, b, limit=120)
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter('always', category=IntegrationWarning)
+        # Take the integral between the two wavelengths
+        intgrl, ierror = quad(c, a, b, limit=120)
+
+        if len(w):
+            log = logging.getLogger(__name__)
+            log.debug(
+                "Integration warning raised in solar radiation calculation"
```

Comment:
Maybe "Integration round off precision warning" or something a little more verbose as to the warning. If I were to just see this warning without knowing what it was, I would think that there was a problem with the integration.

Also, switch to the warning logger since it's more in line with the message.